pkg/util/cloud_prepper.go

```diff
@@ -25,7 +27,24 @@ type CloudPrepper struct {
 	ImagePrepper
 }
 
-func (p CloudPrepper) getFileSystem() (string, error) {
+func (p CloudPrepper) Name() string {
+	return "Cloud Registry"
+}
+
+func (p CloudPrepper) GetSource() string {
+	return p.ImagePrepper.Source
+}
+
+func (p CloudPrepper) SupportsImage() bool {
+	pattern := regexp.MustCompile("^.+/.+(:.+){0,1}$")
+	image := p.ImagePrepper.Source
+	if exp := pattern.FindString(image); exp != image || CheckTar(image) {
```

Comment:
Hmm, why does this support tars?

Reply:
it won't. keeping the functional changes to a minimum in this PR, I fixed this in https://github.com/GoogleCloudPlatform/container-diff/pull/89

Reply:
Ah I see, yeah this worked that way before